.env

```diff
@@ -20,6 +20,7 @@ PROXY_BUILD_CONTEXT=./
 PROXY_PUBLIC_PORT=80
 TIMEZONE=UTC
 GITPOD_ENV=false
+CLIENT_MAX_BODY_SIZE=10M
```

Comment:
I guessed 10M to be a sensible default, probably should be way less than that...

Reply:
@corneliouzbett, I think we can default the max size to 12M.